### PR TITLE
feat(Avatar): add IconAvatar, Avatar

### PR DESCRIPTION
**Pull request by liuderchi**
add `IconAvatar`, `Avatar` for #9 

Reference: [MCS-LITE-UI Storybook `Avatar`](https://mcs-lite-ui.netlify.com/?selectedKind=Avatar&selectedStory=Default&full=0&addons=0&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

### CODE REVIEW

File: src/Avatar/styled-components.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Wrapper: React.ComponentType<*> = styled.div`
+  img,
+  svg {
+    display: inline-block;
+    border-radius: 50%;
+    border: 1px solid ${props => props.theme.color.grayBase};
```
Comment (evenchange4) [inline src/Avatar/styled-components.js]:
```suggestion
    border: 1px solid ${({ theme }: ThemeProps) => theme.color.grayBase};
```

File: src/Icons/svgr/IconAvatar.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const IconAvatar = (props: Props) => (
+  <svg
+    fill="currentColor"
+    width="1em"
+    height="1em"
+    viewBox="0 0 300 300"
+    {...props}
+  >
+    <path fill="none" d="M0 0h300v300H0z" />
```
Comment (evenchange4) [inline src/Icons/svgr/IconAvatar.js]: remove this useless path
```diff
- <path fill="none" d="M0 0h300v300H0z" />
```

File: src/Icons/svgr/IconAvatar.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react';
```
Comment (evenchange4) [inline src/Icons/svgr/IconAvatar.js]: We need import all for flow type checking

```suggestion
import * as React from 'react';
```

File: src/Icons/svgr/IconAvatar.js (hunk 2 above)
Comment (liuderchi) [reply src/Icons/svgr/IconAvatar.js]: ✅ done

File: src/Avatar/styled-components.js (hunk 1 above)
Comment (liuderchi) [reply src/Avatar/styled-components.js]: ✅ done

**Pull request merged**